lib/Optimizer/Transforms/CMakeLists.txt

```diff
@@ -44,6 +44,8 @@ add_cudaq_library(OptTransforms
   QuakeSynthesizer.cpp
   RefToVeqAlloc.cpp
   RegToMem.cpp
+  StateDecomposer.cpp
+  StatePreparation.cpp
   PySynthCallableBlockArgs.cpp
```

Comment:
Can you move the PySynth... file name to maintain alphabetical order here? Thanks in advance.

Reply:
done!